deployment/hasura/metadata/actions.graphql

```diff
@@ -25,6 +25,17 @@ type Query {
   ): EffectiveArgumentsResponse
 }
 
+type Query {
+  getActivityTypeScript(
+    missionModelId: ID!
+    activityTypeName: String!
+  ): TypeScriptResponse
+}
+
+type TypeScriptResponse {
```

Comment:
Another slightly pedantic change: would you mind moving `TypeScriptResponse ` to further down in this file?

Looks like this file has this rough ordering:
- Mutations
- Queries
- Types needed for mutations & queries (`*Response` types)
- Scalars

Reply:
I'm not disagreeing with you. I think there is an argument to be made for an alternative "grouping" of each Query/Mutation/Subscription operation with its own relevant types and collapsing shared types elsewhere. Not sure which is more productive, but I know I like to see everything together so its easier to grok

Reply:
To be honest the other PR has the rough ordering just to be consistent:

deployment/hasura/metadata/actions.graphql

Reply:
No strong preference here besides consistency, so feel free to reorder the rest of the actions if you prefer the grouping order.

Reply:
Not worth reordering the whole thing. Moved it to follow the other PR